easybuild/easyblocks/m/mvapich2.py

```diff
@@ -69,14 +67,10 @@ def configure_step(self):
         add_configopts.append('--with-rdma=%s' % self.cfg['rdma_type'])
 
         # enable specific support options (if desired)
-        if self.cfg['withmpe']:
-            add_configopts.append('--enable-mpe')
```

Comment:
you can use `self.log.warning` for warnings (but they'll only end up in the log file)

to raise an error, use `raise EasyBuildError("<msg goes here>")`